rule-beans/src/com/axonivy/ivy/process/element/rule/ProcessElementExtension.java

```diff
@@ -11,7 +11,6 @@
 
 public class ProcessElementExtension implements IExtensibleStandardProcessElementExtension
 {
-  static final String RULE_ACTIVITY = "RuleActivity";
   static final String DECISION_ACTIVITY = "DecisionActivity";

```

Comment:
I'd update the Decision Table to the new extension points: should be a 10 min tasks with the next sexy interfaces